doc/platform-support.appendix.md

```diff
@@ -0,0 +1,11 @@
+# Platform Support {#appendix-platform-support}
+
+Currently, the `x86_64-linux` platform receives the highest level of support, both in terms of CI and of developer attention.
+The `aarch64-linux` plaform is also well supported, with most packages building succesfully in CI.
+
+On Darwin, `aarch64-darwin` receives better support than `x86_64-darwin`.
+
+For a more formal definition of the different platform support tiers, see [RFC046]. Note though that the platform list in [Appendix A] is out of date.
```

Comment:
```suggestion
A more formal definition of different platform support tiers was accepted with [RFC046], though it has not been fully implemented so far.
```

Reply:
it's not clear what hasn't been implemented. I don't want readers to have to figure that out by reading the whole RFC. That's why I mentioned specifically the platform list.

Reply:
```suggestion
A more formal definition of different platform support tiers was accepted with [RFC046], but is not fully implemented so far.
The preliminary platform list in the RFC's [Appendix A] can be used as guidance, but is not up to date.
```

Reply:
That sounds good to me